From 7b21186f607b6632925fb5464beec468372e5a39 Mon Sep 17 00:00:00 2001
From: Dinh Nguyen <dinguyen@kernel.org>
Date: Mon, 12 Mar 2018 09:49:21 -0500
Subject: [PATCH 23/25] arm64: dts: stratix10: disable false USB overcurrent on
 devkit

commit  1752cc0adb37e96e79a8ce8189dbd8d53016a224 from
https://github.com/altera-opensource/linux-socfpga.git

Disable the USB overcurrent condition that is falsely detected on the
devkit.

Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
index 8f421d6..ec18609 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
@@ -113,6 +113,7 @@
 
 &usb0 {
 	status = "okay";
+	disable-over-current;
 };
 
 &i2c1 {
-- 
2.8.1

